modules/nodes-safe-sysctls-list.adoc

```diff
@@ -0,0 +1,118 @@
+// Module included in the following assemblies:
+//
+// * nodes/containers/nodes-containers-sysctls.adoc
+
+:_content-type: REFERENCE
+[id="safe_and_unsafe_sysctls_{context}"]
+= Safe and unsafe sysctls
+
+Sysctls are grouped into _safe_ and _unsafe_ sysctls.
+
+For system-wide sysctls to be considered safe, they must use proper namespacing and there must be proper isolation between pods on the same node. This means that if you set a sysctl for one pod it must not Any sysctl not addeddo any of the following:
```

Comment:
`proper isolation between pods on the same node. `

This is what is meant when we say a sysctl is namespaced. It should be enough to say:

`For system-wide sysctls to be considered safe it must be namespaced.`

If we really want to explain what it means, maybe add something like:

`A namespced sysctl gives us the following features:
- isolation between namespaces (and therefore pods)
-....

`